src/job-helper.js

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright 2023 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+export class JobHelper {
+  #settingsStore;
+
+  static JOB_STATUS = Object.seal({
+    RUNNING: 'RUNNING',
+    STOPPED: 'STOPPED',
+    COMPLETE: 'COMPLETE',
+  });
+
+  /**
+   * @param {import("./settings.js").SettingsObject} sourceSettings
+   * @param {string} jobId
+   * @returns {boolean} True if the jobId is a full job, matches the
+   *  currentJobId and is in a RUNNING state.
+   */
+  static isCurrentRunningJob(sourceSettings, jobId) {
+    return (
+      jobId.startsWith('FULL::')
```

Comment:
nitpick: declare as a static variable at the top (ex: `JobHelper.JOB_TYPE.FULL`). Also, why are the colons needed?